documentation/setup/download.mdx

```diff
@@ -21,6 +21,7 @@ If you are not sure what type of Home Assistant you are using or how to find tha
 1. Start the SSH add-on
 1. Connect to the SSH add-on
 1. Run the HACS download script
+1. After the script completes, restart Home Assistant
```

Comment:
This does not only apply to OS/Supervised, right? Container installs need to restart their container and Core install need to restart as well.